### PR TITLE
Add Docker image for building contracts.

### PR DESCRIPTION
**Pull request by olonho**

### CODE REVIEW

File: contact-builder/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM rust
```
Comment (ailisp) [inline contact-builder/Dockerfile:1]: Need to pin a specific rust version, such as `rust:1.44.1`
 https://hub.docker.com/_/rust

**Comment by nearmax** [comment]
Let's push it to Docker hub.

**Comment by olonho** [comment]
@nearmax nearprotocoldocker/contractbuild:d2524 on the Dockerhub.

**Comment by ailisp** [comment]
Looks push didn't success (i cannot pull) i'll repush to nearprotocol/contract-builder

**Pull request merged**